doc/langref.html.in

```diff
@@ -10308,14 +10308,568 @@ fn readU32Be() u32 {}
+          <td>
+            The boolean operator {#syntax#}and{#endsyntax#}.
+            <ul>
+              <li>See also {#link|Operators#}</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>{#syntax#}anyframe{#endsyntax#}</pre>
+          </td>
+          <td>
+            {#syntax#}anyframe{#endsyntax#} can be used as a type for variables which hold pointers to function frames.
+            <ul>
+              <li>See also {#link|Async Functions#}</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>{#syntax#}anytype{#endsyntax#}</pre>
+          </td>
+          <td>
+            Function parameters can be declared with {#syntax#}anytype{#endsyntax#} in place of the type. The parameter type will be inferred where the function is called.
+            <ul>
+              <li>See also {#link|Function Parameter Type Inference#}</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>{#syntax#}asm{#endsyntax#}</pre>
+          </td>
+          <td>
+            {#syntax#}asm{#endsyntax#} begins an inline assembly expression. This allows for directly controlling the machine code generated on compilation.
+            <ul>
+              <li>See also {#link|Assembly#}</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>{#syntax#}async{#endsyntax#}</pre>
+          </td>
+          <td>
+            {#syntax#}async{#endsyntax#} can be used before a function call to get a pointer to the function's frame when it suspends.
+            <ul>
+              <li>See also {#link|Async Functions#}</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>{#syntax#}await{#endsyntax#}</pre>
+          </td>
+          <td>
+            {#syntax#}await{#endsyntax#} can be used to suspend the current function until the frame provided after the {#syntax#}await{#endsyntax#} completes. {#syntax#}await{#endsyntax#} copies the value returned from the target function's frame to the caller.
+            <ul>
+              <li>See also {#link|Async Functions#}</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>{#syntax#}break{#endsyntax#}</pre>
+          </td>
+          <td>
+            {#syntax#}break{#endsyntax#} can be used to exit a loop before iteration completes naturally.
+            It can also be used with a block label to return a value from the block.
+            <ul>
+              <li>See also {#link|blocks#}, {#link|while#}, {#link|for#}</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>{#syntax#}catch{#endsyntax#}</pre>
+          </td>
+          <td>
+            {#syntax#}catch{#endsyntax#} can be used to provide a default value for an error union, or capture its error value in an expression.
```

Comment:
Using "default value" here is confusing, this should be described as control flow like `if`.